_ml-commons-plugin/model-serving-framework.md

```diff
@@ -21,7 +21,7 @@ To upload a custom model to OpenSearch, you need to prepare it outside of your O
 
 ### Model support
 
-As of OpenSearch 2.4, the model-serving framework only supports text embedding models without GPU acceleration.
+As of OpenSearch 2.6, the model-serving framework supports text embedding and sentence transformer models. 
```

Comment:
"sentence transformer" is just a kind of text embedding model. Let's just use text embedding models here?